pkg/gktest/result.go

```diff
@@ -26,10 +56,27 @@ type TestResult struct {
 	// If defined, CaseResults is empty.
 	Error error
 
+	// Runtime is the time it took for this Suite of tests to run.
```

Comment:
"this Suite of tests to run": use the appropriate naming.

Reply:
Done.